### ISSUE TITLE
Speech signal condensation

### ISSUE THREAD

**Issue by pedrohenriqp** [snu-mllab/Efficient-Dataset-Condensation#4]
Hello, first of all, congratulations on the work.

I'm trying to reproduce the results of speech signals and have some questions.

1)It seems that only spectrograms were used as the audio representation, am I correct? Was there any testing done with mel spectrograms?
2)Once you have obtained the condensed spectrograms, how do you return to the time domain? Is it necessary to perform the decoding?How does it work?

Thanks in advance

**Comment by Janghyun1230** [comment]
Hello!

1) Yes, linear log-magnitude spectrograms were used. I hadn't tried experiments using mel spectrograms. 
2) I remember that it was obtained through the Griffin-Lim function of the Librosa package.

**Comment by pedrohenriqp** [comment]
Hello, thank you for your kind reply.
To reconstruction the speech signal through the Griffin-Lim, did you use directly the condensed spectrograms in lower resolution (e.g 64 x 64) or is it necessary to use interpolation to up sample to get original resolution? Any recommendation to reproduce the samples obtained [here](https://drive.google.com/drive/folders/1LgVwWJcLGr8PfGzGSwWrebTGkP20oP4S)?

**Comment by Janghyun1230** [comment]
I first upsampled the spectrograms to the original size by using the PyTorch interpolate function and then apply the Griffin-Lim algorithm.

**Comment by pedrohenriqp** [comment]
@Janghyun1230 I am training the condensed speech obtained with ConvNet in an EfficientNet and SquezeeNet, but I am unable to achieve good accuracy. In the case of speech signals, the paper shows results with data obtained and tested using the same architecture, right?  Did you obtain any results by testing on different architectures for the case of speech data?

**Comment by Janghyun1230** [comment]
Yes, the results in the paper used the same architecture in training and testing. We did not experiment with different architectures on speech data. In general, cross-architecture testing tends to suffer from low performance.

**Comment by pedrohenriqp** [comment]
Do you have any ideia of how significant this performance drop would be? For example, training a convnet with the [condensed speech data](https://drive.google.com/drive/folders/1LgVwWJcLGr8PfGzGSwWrebTGkP20oP4S) could achieve an test accuracy of around 80%. However, when training an EfficientNet or SqueezeNet with the same data, the test accuracy drops to around 30%. Any ideas of what might be happening? In the case of speech, would the condensation process be highly dependent on the architecture used to obtain the condensed data?

**Comment by Janghyun1230** [comment]
In the case of speech classification, I speculate overfitting to specific patterns can more easily occur than image data. Currently, more research and analysis is needed on that issue.
